docs/reference-use-cases.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Cluster API Reference Use Cases
+authors:
+  - "@vincepri"
+  - "@ncdc"
+  - "@detiber"
```

Comment:
I'd be partial to say get rid of the authors here, since I'm not sure we have accountability for all that have contributed directly to the text here. I know many of the use cases have been written up by Leah

Reply:
Copy paste ftw